recipes/spirv-tools/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+cmake_minimum_required(VERSION 2.8.12)
+project(test_package)
+
+if( ${BUILD_C_TEST_PACKAGE} )
```

Comment:
```cmake
if( ${BUILD_C_TEST_PACKAGE} )
    set(CMAKE_C_STANDARD 11)
else()
    set(CMAKE_CXX_STANDARD 11)
endif()

set(CMAKE_VERBOSE_MAKEFILE TRUE)
```

All of this can be removed now.